tests/common/common.py

```diff
@@ -158,6 +163,10 @@ def test_receive_unicode_log(self):
         self.assertIsNotNone(found_log, "expected unicode log not found")
 
     def test_monitored_resource(self):
+        if self.language == 'java':
+            # TODO: implement in java
```

Comment:
please open a bug in java repo

Reply:
https://github.com/googleapis/env-tests-logging/issues/56